packages/create-cloudflare/src/__tests__/deploy.test.ts

```diff
@@ -60,6 +60,39 @@ describe("deploy helpers", async () => {
 			expect(wranglerLogin).not.toHaveBeenCalled();
 		});
 
+		test("project is undeployable (complex binding)", async () => {
+			const ctx = createTestContext();
+			// Can't deploy things with bindings (yet!)
+			vi.mocked(readFile).mockReturnValue(`
+				experimental_assets = { directory = "./dist", binding = "ASSETS" }
+
+				[[durable_objects.bindings]]
```

Comment:
Arguably Durable Objects can be deployed if they have migrations setup.
But the previous state of affairs didn't support that, so it is out of scope for this PR to change that.